umm-spec-lib/src/cmr/umm_spec/util.clj

```diff
@@ -386,3 +386,39 @@
   (if sanitize?
     (util/trunc (with-default s sanitize?) n)
     s))
+
+(def key-split-string
```

Comment:
This namespace is out of control. It is already 388 lines long without the new changes. All the new changes you added are specific for parsing ISO distributed urls. I am wondering why you don't put these defs and function in the `cmr.umm-spec.xml-to-umm-mappings.iso-shared.distributions-related-url`. Are we expecting to use these somewhere else other than ISO parsing?

Reply:
I just took the function that you and Daniel were using for parsing orbit strings in `umm-lib` and it seemed like something that could be used in several different places so I put it in the util namespace. I can re-name or move if you think it would make more sense that way

Reply:
I think we should put these defs to `cmr.umm-spec.xml-to-umm-mappings.iso-shared.distributions-related-url` until we really want to share these in other places. And even when that happens, we will probably find that these defs belong to a different namespace than the `util` namespace.